contracts/package.json

```diff
@@ -19,13 +21,15 @@
     "bn.js": "^5.2.0",
     "cairo-ls": "^0.0.4",
     "chai": "^4.3.4",
+    "ethereum-waffle": "^3.4.4",
```

Comment:
Is this used anywhere?

Reply:
I don't think so, removing it in subsequent review PR

Reply:
Ah, it is required after all. It brings the Ethereum specific chai matchers. We use them to assert on errors in tests.